ec2/spark_ec2.py

```diff
@@ -423,13 +426,14 @@ def get_spark_ami(opts):
         b=opts.spark_ec2_git_branch)
 
     ami_path = "%s/%s/%s" % (ami_prefix, opts.region, instance_type)
+    reader = codecs.getreader("ascii")
     try:
-        ami = urlopen(ami_path).read().strip()
-        print("Spark AMI: " + ami)
+        ami = reader(urlopen(ami_path)).read().strip()
```

Comment:
We had a recent discussion on the dev list about dropping Java 6 support. I think Python 2.6 will be due for a similar discussion later this year or early next.